.github/workflows/_calculate-docker-image.yml

```diff
@@ -39,3 +39,7 @@ jobs:
           # longer term fix would be to refactor linter so that building the new
           # image and running lint are done in the same runner like _linux-build
           skip_push: false
+
+      - name: Teardown Linux
+        uses: pytorch/test-infra/.github/actions/teardown-linux@main
+        if: always()
```

Comment:
I'm not sure we want that, as it is also called as part of the build, isn't it?

Reply:
Actually, it's current called only from lint.yml (rest are calling action), so should we just fold it in there?

Reply:
Ah no, that is https://github.com/pytorch/pytorch/blob/main/.github/actions/calculate-docker-image/action.yml GHA while this one is https://github.com/pytorch/pytorch/blob/main/.github/workflows/_calculate-docker-image.yml workflow.  The workflow one is a wrapper used by lint https://github.com/pytorch/pytorch/blob/17737f9d0e683a01b1873e9b7f948de9035e4951/.github/workflows/lint.yml#L17-L21 as a reusable workflow

Reply:
> Actually, it's current called only from lint.yml (rest are calling action), so should we just fold it in there?

Sound good.  Let me see if I can move it into lint and clean up the workflow